src/test/java/com/snowflake/kafka/connector/internal/LoggerHandlerTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2019 Snowflake Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.snowflake.kafka.connector.internal;
+
+import org.junit.Test;
+
+public class LoggerHandlerTest {
+  @Test
+  public void testLogMessage() {
+    String name = "test.logger.name";
+    String msg = "super useful logging message";
+
+    LoggerHandler loggerHandler = new LoggerHandler(name);
+    loggerHandler.info(msg);
+    // TODO @rcheng: question - how to test that the log was correct? since we dont have a mock
```

Comment:
can we test this? Is there a way to catch the logs being emitted, or a way to mock the Logger?

Reply:
https://stackoverflow.com/a/1828268

Reply:
thanks! added testing